weaver/core/network/corda-interop-app/interop-workflows/src/test/kotlin/org/hyperledger/cacti/weaver/imodule/corda/HandleExternalRequestTests.kt

```diff
@@ -127,7 +127,8 @@ class HandleExternalRequestTests {
         network.runNetwork()
         val invalidSignatureError = invalidSignatureFuture.getOrThrow()
         assertTrue(invalidSignatureError.isLeft())
-        assertEquals("Verification Error: Error verifying signature: signature length is wrong", invalidSignatureError.fold({ it.message }, { "" }))
+        /* assertEquals("Verification Error: Error verifying signature: signature length is wrong", invalidSignatureError.fold({ it.message }, { "" })) */
+        assertEquals("Signature Verification Error for certificate for O=PartyA, L=London, C=GB", invalidSignatureError.fold({ it.message }, { "" }))
```

Comment:
Why are these attributes hardcoded?

Reply:
Actually all the tests inputs were hardcoded here, so that's why this also.